core/auth/controller.go

```diff
@@ -0,0 +1,36 @@
+package auth
+
+import (
+	"context"
+
+	"flamingo.me/flamingo/v3/framework/web"
+)
+
+type WebDecorator struct {
+	wis *WebIdentityService
+}
+
+func (wd *WebDecorator) RequireIdentified(action web.Action) web.Action {
```

Comment:
Probably we can remove some methods from security module inside app service and middleware, which are doing similar job.